masterlist.yaml

```
@@ -15869,11 +15869,29 @@ plugins:
         itm: 122
 
   # Additional Clockwork
+  - name: 'ClockworkBugFixes.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+    clean:
+      - crc: 0x6ED1EE37
+        util: 'SSEEdit v4.0.3'
```

xEdit v4.0.4 was recently released, so that should be used for new cleaning reports.

```suggestion
    clean:
      - crc: 0x6ED1EE37
        util: 'SSEEdit v4.0.4'
```